protos/local/workflow_activation.proto

```diff
@@ -42,6 +40,14 @@ message WFActivationJob {
         SignalWorkflow signal_workflow = 7;
         /// An activity was resolved with, result could be completed, failed or cancelled
         ResolveActivity resolve_activity = 8;
+
+        /// Remove the workflow identified by the [WFActivation] containing this job from the cache
+        /// after performing the activation.
+        ///
+        /// If other job variant are present in the list, this variant will be the last job in the
+        /// job list. The boolean value is irrelevant, since the variant type is what matters. It
+        /// will be set to true if this is the variant.
+        bool remove_from_cache = 50;
```

Comment:
This is a weird place for this flag. Why is this a different type of WFActivationJob? Or at least a flag on WFActivation?

Reply:
It is a different type of job - it's inside the oneof here for what the job is.